test/models/bot/alegre_test.rb

```diff
@@ -129,7 +122,7 @@ def setup
     pm2 = create_project_media project: p
     pm3 = create_project_media project: p
     create_relationship source_id: pm3.id, target_id: pm2.id
-    @bot.add_relationships(pm1, [pm2.id])
+    Bot::Alegre.add_relationships(pm1, [pm2.id])
     r = Relationship.last
     assert_equal pm1, r.target
     assert_equal pm3, r.source
```

Comment:
Some tests still need to be implemented... the following lines of `app/models/bot/alegre.rb` are not covered: 7-8, 55-57, 61, 130-131